source/macos/AdaptiveCards/AdaptiveCards/Views/ACRTextField.swift

```diff
@@ -0,0 +1,104 @@
+import AdaptiveCards_bridge
+import AppKit
+
+class ACRTextField: NSTextField {
+    var isDarkMode: Bool = false
+    
```

Comment:
```suggestion
    private let renderConfig: RenderConfig
    
```
We can store the renderConfig instead and use it in all places.
Actually preferably `InputFieldConfig`.